core/kits/documents/tabs/colors-and-typography.php

```diff
@@ -35,23 +39,193 @@ public function register_tab_controls() {
 			'title',
 			[
 				'type' => Controls_Manager::TEXT,
+				'label_block' => true,
 			]
 		);
 
 		$repeater->add_control(
 			'color',
 			[
 				'type' => Controls_Manager::COLOR,
+				'label_block' => true,
 				'dynamic' => [],
+				'selectors' => [
+					'{{WRAPPER}}' => '--e-global-color-{{_id.VALUE}}: {{VALUE}}',
+				],
+			]
+		);
+
+		$default_colors = [];
+
+		if ( Plugin::$instance->kits_manager->is_custom_colors_enabled() ) {
```

Comment:
Due to the new terminology - can you rename the two methods to `is_system_colors_enabled`, `is_system_typography_enabled`.